deploy/000_deploy_postage.ts

```diff
@@ -1,29 +1,38 @@
-import { HardhatRuntimeEnvironment } from 'hardhat/types';
 import { DeployFunction } from 'hardhat-deploy/types';
+import { networkConfig, developmentChains, deployedBzzData } from '../helper-hardhat-config';
 
-const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
-  const { deployments, getNamedAccounts } = hre;
-  const { deploy, execute, read } = deployments;
+const func: DeployFunction = async function ({ deployments, getNamedAccounts, network, ethers }) {
+  const { deploy, log } = deployments;
+  const { deployer } = await getNamedAccounts();
 
-  const { deployer, oracle, redistributor } = await getNamedAccounts();
+  let token = null;
```

Comment:
```suggestion
  let token: Contract | null = null;
```
use types whenever you can and don't use the same variable for different types when it is not justified.